apps/smartdoc/conf.py

```diff
@@ -111,7 +111,7 @@ def get_db_setting(self) -> dict:
             "ENGINE": self.get('DB_ENGINE'),
             "POOL_OPTIONS": {
                 "POOL_SIZE": 20,
-                "MAX_OVERFLOW": 5
+                "MAX_OVERFLOW": 80
             }
         }
 
```

Comment:
The provided code has a minor issue with the `MAX_OVERFLOW` setting. The current value is 5, but you've suggested an increase to 80.

Suggestion: Consider increasing the `MAX_OVERFLOW` to accommodate potential spikes in database usage. However, adjust this based on your specific application's needs and how often database queries might spike. A higher value means that more connections can be opened when needed, which ensures efficient use of resources without frequent connection creation/destruction cycles.